src/js/base/compat.js

```diff
@@ -90,9 +97,12 @@ if (
         typeof __webpack_require__ === "undefined" &&
         (typeof navigator === "undefined" || navigator.product !== "ReactNative")
     ) {
-        fetch = require("node-fetch");
-    } else {
+        fetch = require("node-fetch").default;
+    } else if (typeof window !== "undefined") {
         fetch = window.fetch;
+        // eslint-disable-next-line camelcase
+    } else if (typeof __webpack_require__ !== "undefined") {
+        fetch = require("node-fetch").default;

```

Comment:
__webpack_require__

Reply:
@gcandal Do you mean doing `fetch = __webpack_require__("node-fetch").default;`?

Reply:
yup!